src/httpserver.cpp

```diff
@@ -21,12 +21,15 @@
 #include <util/threadnames.h>
 #include <util/translation.h>
 
+#include <condition_variable>
 #include <cstdio>
 #include <cstdlib>
 #include <deque>
 #include <memory>
+#include <mutex>
```

Comment:
can be removed.

Reply:
+1, and we're missing `#include <event2/http.h>` (for `evhttp_request`)

Reply:
Both done